input/pagecontent/StructureDefinition-ncpi-family-relationship-intro.md

```diff
@@ -0,0 +1,19 @@
+#### Key Guidelines
+Family Relationships describe the relationship between two Participants. The core use case is to present biological parentage of a participant to support family / pedigree analyses. In this spirit, platforms should seek to provide minimally the information in a “ped” file: if known, a Participant should have a Family Relationship to their biological mother and father. Twins are also a high priority item for reporting. Further extended relationships can be made available using Family Relationship, but may not be as widely supported as they are harder to interpret.
+
+##### Added Profile Restrictions
+In order to ensure that our resources are interoperable across studies, we have employed a number of restrictions that should make consuming Patient resources more consistent.
+
+* ID **should** be a globally unique identifier associated with the person. This practice is intended to make constructing queries for the same person compatible across different servers (such as QA vs PROD) but also to make the resource URLs more meaningful. 
+
+
+#### Recommended Practices
+TODO: Write Recommended Practices
+
+##### FHIR Mappings
+The following fields from the shared data model are to be mapped into the NCPI Participant as shown below:
+
+| **Logical Model Property** | **Cardinality** |  **NCPI Person Mapping** |**Usage Guidance** | **Notes** |
+Subject|1..1|Reference| Required | |
```

Comment:
We need the actual fields where these are mapped to within FHIR. So, in this case, it would be subject, but the target would map to focus, etc. I think the fact that it will be a reference is unnecessary since that's the data type of subject and focus. If you want to be more explicit, you could add that detail in notes, saying something like "Target is indicated as a reference to the participant in the focus field." but I'm not sure that's necessary.

Reply:
I see. Thanks for catching that.